packages/ppx_deriving_yaml/ppx_deriving_yaml.0.1.0/opam

```diff
@@ -15,7 +15,7 @@ depends: [
   "ocaml" {>= "4.08.1"}
   "ppxlib" {>= "0.14.0"}
   "yaml"
```

Comment:
```suggestion
  "yaml" {< "3.0.1"}
```
As i understand it, the ppx itself generates code that uses Rresult, so this should be on the dependency overall, not just for tests

Reply:
Fair enough. In principle the ppx works with newer versions, you just need to manually link agains rresult